### PR TITLE
Create scivision.yml

### PR DESCRIPTION
**Pull request by edwardchalstrey1**
@ots22 can you merge this and give me rights for this repo? I've also put it here: https://github.com/alan-turing-institute/intake-plankton <- we probably don't need two separate repos so I may delete the other one (just a fork of something Scott made)

### CODE REVIEW

**Comment by ots22** [comment]
You have admin rights now.

The current dataset has changed in format slightly, so this won't be quite right any more.  Are you okay to attempt to modify it? I think @acocac was trying to get the new dataset mounted on Azure so you can try it out.

**Comment by acocac** [comment]
@ots22, @edwardchalstrey1 and I yesterday mounted the new dataset using a SAS token on Azure, see `Plankton-cefas-blob` in the AML dashboard.

**Comment by edwardchalstrey1** [comment]
@acocac do you have a modified version if the intake catalog (`scivision.yml`) which works to load the new data?

**Comment by acocac** [comment]
@edwardchalstrey1, the modified version with the new dataset should be (note the dataset should be mounted into a directory named `data`):

```
sources:
  plankton_single:
      description: Load a single labeled images from CEFAS zooplankton dataset
      origin: 
      driver: intake_xarray.image.ImageSource
      parameters:
        id:
          description: which filename
          type: str
          default: Pia1.2017-10-03.1726+N00296780_hc
      args:
        urlpath: './data/images/{{id}}.tif'
        storage_options: {'anon': True}
        exif_tags: True

  plankton_multiple:
      description: Labeled images from CEFAS zooplankton dataset
      origin: 
      driver: intake_xarray.image.ImageSource
      args:
        urlpath: './data/images/{filename}.tif'
        chunks: {}
        storage_options: {'anon': True}
        coerce_shape: [256, 256]
        exif_tags: True
```

**Comment by acocac** [comment]
@ots22 @edwardchalstrey1 bear in mind, we can also add an entry to the intake catalogue to load the csv file...

**Review by acocac — CHANGES_REQUESTED** [review]
@edwardchalstrey1, find below the additional entry to also load the `index.csv` file from the YML file.

```
  index_file:
      description: Contains the images index, filename, classification levels
      origin: 
      driver: csv
      args:
        urlpath: './data/index.csv'
```

File: scivision.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  plankton_multiple:
+      description: Labeled images from CEFAS zooplankton dataset
+      origin: 
+      driver: intake_xarray.image.ImageSource
+      args:
+        urlpath: './data/images/{filename}.tif'
+        chunks: {}
+        storage_options: {'anon': True}
+        coerce_shape: [256, 256]
```
Comment (ots22) [inline scivision.yml]: Coerce to the maximum image size, so we don't crop any image
```suggestion
        coerce_shape: [832, 1040]
```

Comment (ots22) [reply scivision.yml]: There isn't really a better way than hard-coding the size for now. We can leave it to the participants to resize/rescale the images if they want to.

**Comment by ots22** [comment]
This all looks good to me - okay to merge with the above change?

**Pull request merged**